### PR TITLE
Update ingester.md

### PR DESCRIPTION
**Pull request by willychrisza**
minor changes for clarity

#### Which issue(s) this PR fixes or relates to

Fixes #<issue number>

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by dimitarvdimitrov — APPROVED** [review]

**Review by osg-grafana — CHANGES_REQUESTED** [review]
Needs doc review

**Comment by dimitarvdimitrov** [comment]
the changes seem ok and correct, but I'm not sure if we want to break the consistency with the other components - the title is `Grafana Mimir <component>` whereas this changes it to `Understanding <component>`.

Also if we do decide to go with the new format, it will be great if we don't break existing links.

File: docs/sources/mimir/operators-guide/architecture/components/ingester.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ description: "The ingester writes incoming series to long-term storage."
 weight: 30
 ---
 
-# Grafana Mimir ingester
+# Understanding the Grafana Mimir Ingester Component
 
 The ingester is a stateful component that writes incoming series to [long-term storage]({{< relref "../about-grafana-mimir-architecture/index.md#long-term-storage" >}}) on the write path and returns series samples for queries on the read path.
 
-Incoming series from [distributors]({{< relref "distributor.md" >}}) are not immediately written to the long-term storage but are either kept in ingesters memory or offloaded to ingesters disk.
+Incoming time series data from [distributors]({{< relref "distributor.md" >}}) is temporarily stored in the ingester's memory or offloaded to disk before being written to long-term storage.
```
[the remark this answers is not in the document]
Comment (osg-grafana) [reply docs/sources/mimir/operators-guide/architecture/components/ingester.md:12]: As odd as it sounds, "data" is plural.

File: docs/sources/mimir/operators-guide/architecture/components/ingester.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ description: "The ingester writes incoming series to long-term storage."
 weight: 30
 ---
 
-# Grafana Mimir ingester
+# Understanding the Grafana Mimir Ingester Component
```
Comment (osg-grafana) [inline docs/sources/mimir/operators-guide/architecture/components/ingester.md:8]:
```suggestion
# Grafana Mimir ingester
```
The title must match the metadata, and the other components have parallel structure to the original. Reverting.

File: docs/sources/mimir/operators-guide/architecture/components/ingester.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (willychrisza) [reply docs/sources/mimir/operators-guide/architecture/components/ingester.md:12]: oh gosh, i did know this once

File: docs/sources/mimir/operators-guide/architecture/components/ingester.md [hunk 3]
```diff
@@ -37,11 +37,11 @@ Ingesters store recently received samples in-memory in order to perform write de
 If the ingesters immediately write received samples to the long-term storage, the system would have difficulty scaling due to the high pressure on the long-term storage.
 For this reason, the ingesters batch and compress samples in-memory and periodically upload them to the long-term storage.
 
-Write de-amplification is the main source of Mimir's low total cost of ownership (TCO).
+Write de-amplification is a key factor in reducing Mimir's total cost of ownership (TCO).
 
 ## Ingesters failure and data loss
 
-If an ingester process crashes or exits abruptly, all the in-memory series that have not yet been uploaded to the long-term storage could be lost.
+If an ingester process crashes or exits abruptly, any in-memory time series data that has not yet been uploaded to long-term storage may be lost.
```
Comment (osg-grafana) [inline docs/sources/mimir/operators-guide/architecture/components/ingester.md:44]:
```suggestion
If an ingester process crashes or exits abruptly, any in-memory time series data that have not yet been uploaded to long-term storage may be lost.
```

File: docs/sources/mimir/operators-guide/architecture/components/ingester.md [hunk 4]
```diff
@@ -63,7 +63,8 @@ If multiple ingesters fail, time series might be lost if the failure affects all
 The write-ahead log (WAL) writes all incoming series to a persistent disk until the series are uploaded to the long-term storage.
 If an ingester fails, a subsequent process restart replays the WAL and recovers the in-memory series samples.
 
-Contrary to the sole replication, and given that the persistent disk data is not lost, in the event of the failure of multiple ingesters, each ingester recovers the in-memory series samples from WAL after a subsequent restart.
+Unlike sole replication, the WAL ensures that in-memory time series data is not lost in the event of multiple ingester failures. Each ingester can recover the data from the WAL after a subsequent restart.
```
Comment (osg-grafana) [inline docs/sources/mimir/operators-guide/architecture/components/ingester.md:66]:
```suggestion
Unlike sole replication, the WAL ensures that in-memory time series data are not lost in the case of multiple ingester failures. Each ingester can recover the data from the WAL after a subsequent restart.
```
"event" is overloaded in our industry, so changing word to "case".

**Comment by pracucci** [comment]
Thanks for contributing to doc! I committed the Ursula's edits to move forward with this PR. @osg-grafana can you take a final look, please?

**Pull request merged**